Makefile

```diff
@@ -50,20 +50,27 @@ worker-key-pair: ca
 	openssl req -new -sha256 -key /etc/osbuild-composer/worker-key.pem	-out /etc/osbuild-composer/worker-csr.pem -subj "/CN=localhost"
 	openssl x509 -req -in /etc/osbuild-composer/worker-csr.pem  -CA /etc/osbuild-composer/ca-crt.pem -CAkey /etc/osbuild-composer/ca-key.pem -CAcreateserial -out /etc/osbuild-composer/worker-crt.pem
 
-.PHONY: rpm
-rpm:
-	mkdir -p $(CURDIR)/rpmbuild/SPECS $(CURDIR)/rpmbuild/SOURCES
 
-	echo "%global commit $(COMMIT)" | cat - golang-github-osbuild-composer.spec \
-		> rpmbuild/SPECS/golang-github-osbuild-composer-$(COMMIT).spec
 
-	git archive --prefix=osbuild-composer-$(COMMIT)/ --format=tar.gz HEAD \
-		> rpmbuild/SOURCES/osbuild-composer-$(COMMIT).tar.gz
+RPM_SPECFILE=rpmbuild/SPECS/golang-github-osbuild-composer-$(COMMIT).spec
+RPM_TARBALL=rpmbuild/SOURCES/osbuild-composer-$(COMMIT).tar.gz
+
+$(RPM_SPECFILE):
+	mkdir -p $(CURDIR)/rpmbuild/SPECS
+	echo "%global commit $(COMMIT)" | cat - golang-github-osbuild-composer.spec > $(RPM_SPECFILE)
```

Comment:
Could we pull the spec file from git HEAD? Otherwise there could be local edits in the specfile, but not in the sources. Odd?